docs/website/docs/getting-started/quickstart.md

```diff
@@ -1,163 +1,226 @@
+
+
+For our last step, we will now link the backend Java component (which also uses the Postgres service) and the frontend Node.JS component.
+
+This will allow both to communicate with each other in order to store persistent data.
+
+
+Change to the `frontend` component directory and link it to the backend:
 ```shell
-$ odo url list
-Found the following URLs for component backend
-NAME         STATE      URL                                       PORT     SECURE     KIND
-8080-tcp     Pushed     http://8080-tcp.192.168.39.117.nip.io     8080     false      ingress
+cd ../frontend
+odo link backend
 ```
-In this case, the URL to load in browser would be `http://8080-tcp.192.168.39.117.nip.io/api/v1/todos`. Note that the URL would be different in your case depending on what the minikube VM's IP is. When you load the URL in the browser, you should see an empty list:
+
+Push the changes:
 ```shell
-[]
+odo push
 ```
 
-## Create the frontend component
+We're done! Now it's time to test your new multi-component and service application.
 
-Our frontend component is a React application that communicates with the backend component. Create the frontend component:
+## Testing your application
 
-```sh
-cd ../frontend
-odo create nodejs frontend
-odo url create --port 3000 --host `minikube ip`.nip.io
-odo push
-```
+### Frontend Node.JS component
 
-Open the URL for the component in the browser, but note that you won't be able to add, remove or list the todos yet because we haven't linked the frontend and the backend components:
+Find out what URL is being used by the frontend:
 ```shell
 odo url list
+Found the following URLs for component frontend
+NAME          STATE      URL                           PORT     SECURE     KIND
+http-3000     Pushed     http://<URL-OUTPUT>           3000     false      ingress
 ```
 
-## Link the frontend and backend components
+Visit the link and type in some to-dos!
+
+
+### Backend Java component
 
-To link the frontend component to backend:
+Let's see if each to-do is being stored in the backend api and database.
 
+Find out what URL is being used by the backend:
 ```shell
-odo link backend
-odo push
+odo url list
+Found the following URLs for component backend
+NAME         STATE      URL                                       PORT     SECURE     KIND
+8080-tcp     Pushed     http://<URL-OUTPUT>                       8080     false      ingress
 ```
 
-Now reload the URL of frontend component and try adding and removing some todo items. The list of items appears by default on the same page just below the input box that reads `Add a new task`.
\ No newline at end of file
+When you `curl` or view the URL on your browser, you'll now see the list of your to-dos:
+
+```yaml
+curl http://<URL-OUTPUT>/api/v1/todos
+[{"id":1,"description":"hello"},{"id":2,"description":"world"}]
+```
+
+## Further reading
+
+Want to learn what else `odo` can do? Check out the [Tutorials](/docs/intro) on the sidebar.
```

Comment:
But the link is to "Introduction" page. "Tutorials" is a section (not a page), so adding a link to it would lead to 404.

Reply:
What's the need? The "Tutorials" section on the sidebar is pretty clear, no? User can use their mouse. 😉